README.md

```diff
@@ -227,48 +227,20 @@ end
 
 #### Good
 
-Instead, validate it in a separate migration with a more agreeable `RowShareLock`. This approach is documented by Postgres to have “[the least impact on other work](https://www.postgresql.org/docs/current/sql-altertable.html).”
```

Comment:
The readme change looks great, but won't be able to update it until the new release is out. Can you back out this change and paste the updated markdown as a comment?

Reply:
To work around this, you can use `add_foreign_key_concurrently` helper method. It ensures that no downtime
is needed. It will create foreign key and validate it separately with a more agreeable `RowShareLock`.
This approach is documented by Postgres to have “[the least impact on other work](https://www.postgresql.org/docs/current/sql-altertable.html).”

```ruby
class AddForeignKeyOnUsers < ActiveRecord::Migration[6.0]
  def change
    add_foreign_key_concurrently :users, :orders
  end
end
```